pkgs/desktops/gnome-3/extensions/timepp/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "gnome-shell-extension-timepp-${version}";
+  version = "440cf85dc68d9e6ba876793f13910ee6239622cf";
```

Comment:
please you the date of the commit instead of a commit hash as version number

Reply:
Done.